eval/Makefile

```
@@ -47,4 +79,8 @@ clean-problems: clean-vo
 	rm -f $(patsubst problems/%.v,problems/%.v.bak,$(FFILES))
 	rm -f $(patsubst problems/%.v,problems/%.glob,$(FFILES))
 
-.PHONY: clean clean-vo clean-problems check atp reconstr init all
+.PHONY: clean clean-vo clean-problems test test-prepare-corpus test-grid-engine \
+        test-compile-supervisor test-atp-makefile test-premise-grid-spec \
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)